modules/flowable-ui-modeler/flowable-ui-modeler-rest/src/main/java/org/flowable/app/rest/editor/ModelResource.java

```diff
@@ -312,127 +315,23 @@ protected ModelRepresentation createNewModel(String name, String description, In
     return new ModelRepresentation(newModel);
   }
 
-  @RequestMapping(value = "/rest/model/exportForDeploy/{modelId}")
+  @RequestMapping(value = "/rest/models/{modelId}/exportForDeploy")
```

Comment:
This is not flowable existing mapping, I added this mapping, I changed it to make it more consistent like the existing flowable mapping